Makefile

```diff
@@ -1,11 +1,19 @@
 
-APPNAME = moolticute_ssh-agent
+APPNAME = mc-agent
 
 TAGS = ""
 BUILD_FLAGS = "-v"
 #LDFLAGS = "-H windowsgui" #this prevents opening of console on windows
 
-.PHONY: build clean
+.PHONY: build clean setup
+
+setup:
+	go get github.com/gorilla/websocket
```

Comment:
Why adding this rule?
`go get github.com/raoulh/mc-agent` will pull all dependencies. and `go get -u github.com/raoulh/mc-agent` will update all of them. There is no need for this rule. Also it forces us to keep track of all deps here, which is bad.